ioservice/cob_foms.c

```diff
@@ -489,8 +489,12 @@ static bool cob_pool_version_mismatch(const struct m0_fom *fom)
 		M0_LOG(M0_DEBUG, "cob pver"FID_F", common pver"FID_F,
 				FID_P(&cob->co_nsrec.cnr_pver),
 				FID_P(&common->c_body.b_pver));
-		return !m0_fid_eq(&cob->co_nsrec.cnr_pver,
-				  &common->c_body.b_pver);
+		if (!m0_fid_eq(&cob->co_nsrec.cnr_pver, &common->c_body.b_pver))
+			return true;
```

Comment:
Why don't we call m0_cob_put() in this case?

Reply:
Corrected.

Reply:
OK